user_docs/en/changes.t2t

```diff
@@ -211,6 +211,7 @@ Highlights of this release include  support for charts in Microsoft word and Pow
  - NVDA's braille input functionality can be used with these as well as other BC6 displays with firmware 3.0.0 and above.
 - Early support for Google Sheets with Braille mode enabled. (#7935)
 - Support for Eurobraille Esys, Esytime and Iris braille displays. (#7488)
+- Added support for Foxit Reader and Foxit PhantomPDF. (#8944)

```

Comment:
Could you revert the changes to this file altogether? Changing the changes file will be taken care of before merging this pr.